packages/adapter/tests/dummy/config/targets.js

```diff
@@ -2,12 +2,6 @@
 
 const browsers = ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
```

Comment:
Perhaps we should change this to `last 2`?  Thoughts @igorT ?

Reply:
We should do whatever Ember does

Reply:
```
  browsers = ['last 2 Chrome versions', 'last 2 Firefox versions', 'Safari 12', 'last 2 Edge versions'];
```

Good idea.  Here it is and updated in this PR.